lib/srv/authhandlers.go

```diff
@@ -133,33 +133,29 @@ func (h *AuthHandlers) CheckPortForward(addr string, ctx *ServerContext) error {
 func (h *AuthHandlers) UserKeyAuth(conn ssh.ConnMetadata, key ssh.PublicKey) (*ssh.Permissions, error) {
 	fingerprint := fmt.Sprintf("%v %v", key.Type(), sshutils.Fingerprint(key))
 
-	// as soon as key auth starts, we know something about the connection, so
```

Comment:
are changes in this file relevant to the PR?
if not, please revert

Reply:
required for tests to pass.  fixes a data race that was being triggered by some of the concurrent session tests.